PostBuild.sh

```diff
@@ -407,14 +407,22 @@ function SELsetup {
       err_exit "Running fixfiles in chroot..." NONE
       chroot "${CHROOTMNT}" /sbin/fixfiles -f relabel || \
         err_exit "Errors running fixfiles"
+   # This block *shouldn't* be necessary
    elif [[ $(
       grep -Eq '^SELINUX=(enabled|permissive)' "${CHROOTMNT}/etc/selinux/config"
    )$? -eq 0 ]]
    then
       err_exit "Cofiguring image for relabel-at-boot operation" NONE
       touch "${CHROOTMNT}/.autorelabel" || \
         err_exit "Failed creating /.autorelabel file"
```

Comment:
Any particular reason to keep this block, since the next one is doing the same thing unconditionally?

Reply:
Technically? No. It's what I put there _before_ I realized that the `selinux-policy` RPM was completely dorked and not just sorta-dorked.

Reply:
Up to you either way, I was just curious if you had considered collapsing the two blocks since one is a bit redundant.

Reply:
Mostly, I'd left it as a "note to future self" kind of thing.  Really, if things were working as they _ought_ to be (in Azure), then this entire PR would be superfluous.